docs/v1/api.md

```diff
@@ -1,63 +1,93 @@
 # V1 API
 
-This document contains resource types and their standard usage in v1 service
-catalog. Although this API will be implemented in Kubernetes, other systems
-are not precluded from implementing it as well.
+This document describes the API resource types and usage in the v1 service
+catalog. Although this API will be implemented in a Kubernetes repo, other
+systems are not precluded from implementing it as well.
 
 # Resource Types
 
 This section lists descriptions of the resources in the service catalog API.
 
 *__Note:__ All names herein are tentative, and should be considered placeholders
-and used as placeholders for purposes of discussion only. This note will be
-removed when names are finalized.*
+for purposes of discussion. This note will be removed when names are finalized.*
 
-## `Broker`
+## `Broker` Resource
 
-This resource is created by an administrator to instruct the service catalog's
-controller event loop to do the following:
+A `Broker` represents an entity that provides service classes for use in the
+catalog.
 
-1. Make a request against a given CF service broker's catalog endpoint
+An administrator to creates an instance of the `Broker` resource to indicate
+their intent to consume that broker in the service catalog. 
```

Comment:
instead of consume, I wonder if something that says about intent to make the Service Classes provided by that broker available in the service catalog would be clearer?
their intent to make Service Classes provided by that broker available in the service catalog